mgmt/imgmgr/src/imgmgr.c

```diff
@@ -844,13 +853,47 @@ imgr_upload(struct mgmt_cbuf *cb)
     return imgr_upload_good_rsp(cb);
 }
 
+void
+imgr_dfu_stopped(void)
+{
+    if (imgmgr_dfu_callbacks_fn->dfu_stopped_cb != NULL)
```

Comment:
brackles here and below are not according to https://github.com/apache/mynewt-core/blob/master/CODING_STANDARDS.md